gi2ds 번역 완료

deep-diver opened the pull request:
gi2ds - google images search 2 dataset 사용법에 대한 번역 완료.

원문에 사용된 이미지 등은 실제 repository에 포함된 이미지 URL을 참조함.

deep-diver committed:
Create gi2ds.md

kkweon commented on tools/gi2ds.md:
```diff
@@ -0,0 +1,57 @@
+![gi2ds - Google Image Search to Dataset](https://github.com/toffebjorkskog/ml-tools/blob/master/images/gi2ds-usage.png?raw=true)
+
+## 사용을 시작해 보는법
+[아래쪽에 설명된 북마크](#bookmarklet) 을 사용하거나, 이미지 검색이 이뤄진 페이지에서, 브라우져의 자바스크립트 콘솔을 실행한 후 아래의 자바스크립트 코드를 붙여넣는 두 가지 방법이 있다.
```
```suggestion
[아래쪽에 설명된 북마크릿(bookmarkelt)](#bookmarklet) 을 사용하거나, 이미지 검색이 이뤄진 페이지에서, 브라우져의 자바스크립트 콘솔을 실행한 후 아래의 자바스크립트 코드를 붙여넣는 두 가지 방법이 있다.
```

해주심이 좋을 것 같습니다. 

기술 문서이기 때문에 정확한 용어 사용이 중요하다고 생각됩니다.

deep-diver replied:
제가 북마크릿 이라는 단어를 잘 몰랐는데, 찾아보니 브라우져 상단 아이콘식으로 등록되는 버튼 이었군요. 사람들도 잘 아는 단어 같으니 제안하신대로 '북마크릿'이 더 적당하다는 생각이 듭니다! :)

kkweon commented on tools/gi2ds.md:
```diff
@@ -0,0 +1,57 @@
+# gi2ds - Google Image Search 2 Dataset
+**gi2ds** 는 구글 이미지 검색에 기반하여 이미지 데이터셋을 생성할 때 도움을 줄 수 있도록 디자인 되었다. 이미지를 클릭함으로써 연관성이 적은 이미지를 포함시키거나 제외시킬 수 있게 해 준다. 이미지의 URL 목록은 우측 하단에 표시가 된다. 더 많은 이미지를 얻고 싶다면 화면을 아래로 스크롤링 해서, 더 많은 이미지를 불러오거나 `show more results` 버튼을 클릭한 후 계속해서 스크롤을 내리면 된다.
```
```suggestion
**gi2ds** 는 구글 이미지 검색에 기반하여 이미지 데이터셋을 생성할 때 도움을 줄 수 있도록 디자인되었다. 이미지를 클릭함으로써 연관성이 적은 이미지를 포함하거나 제외할 수 있게 해 준다. 이미지의 URL 목록은 우측 하단에 표시가 된다. 더 많은 이미지를 얻고 싶다면 화면을 아래로 스크롤링해서, 더 많은 이미지를 불러오거나 `show more results` 버튼을 클릭한 후 계속해서 스크롤을 내리면 된다.
```

deep-diver replied:
띄워쓰기 교정 감사합니다! 😊

kkweon reviewed (changes requested):
와우 좋네요.

참고로 리뷰는 one-way 강요가 아니라 디스커션입니당. 
제 생각엔 북마크릿 써주는게 좋을 것 같아요. 어떻게 생각하시는지

deep-diver committed:
Update tools/gi2ds.md

북마크 -> 북마크릿으로 변경

Co-Authored-By: Mo Kweon <kkweon@gmail.com>

deep-diver committed:
Update tools/gi2ds.md

띄워쓰기 교정

Co-Authored-By: Mo Kweon <kkweon@gmail.com>

deep-diver commented:
@kkweon 반영 되었습니다 :)

kkweon commented on tools/gi2ds.md:
````diff
@@ -0,0 +1,57 @@
+<a name="bookmarklet"></a>
+아래의 코드 스니펫을 [북마크 추가하기](images/gi2ds-add-bookmark.png) 에 설명된 방식대로, 추가하여 북마크를 생성한다.
+
+```javascript:(function()%7B(function(e%2C%20s)%20%7Be.src%20%3D%20s%3Be.onload%20%3D%20function()%20%7BjQuery.noConflict()%3BjQuery('%3Cstyle%20type%3D%22text%2Fcss%22%3E%20.remove%20%7B%20opacity%3A0.3%3B%7D%5Cn%20.urlmodal%20%7Bpadding%3A%2010px%3B%20background-color%3A%20%23eee%3B%20position%3A%20fixed%3B%20bottom%3A%200%3B%20right%3A%200%3B%20height%3A%20100px%3B%20width%3A%20300px%3B%20z-in ...
````
코드블락 뒤에 줄바꿈 해주시는게 좋을 것 같아요.

한줄에 다 붙여서 써있어서 인라인으로 렌더링되네요

deep-diver replied:
<img width="1314" alt="Screen Shot 2019-08-15 at 11 33 37 AM" src="https://user-images.githubusercontent.com/26025527/63069088-98998e80-bf50-11e9-9fc4-28f909374501.png">

제  PC 에서는 첨부 그림처럼 보여지긴 하는데, 일단 줄 바꿈을 추가해 놓을게요!

kkweon replied:
블락으로 하면 이렇게 되구요

```
javascript:(function()%7B(function(e%2C%20s)%20%7Be.src%20%3D%20s%3Be.onload%20%3D%20function()%20%7BjQuery.noConflict()%3BjQuery('%3Cstyle%20type%3D%22text%2Fcss%22%3E%20.remove%20%7B%20opacity%3A0.3%3B%7D%5Cn%20.urlmodal%20%7Bpadding%3A%2010px%3B%20background-color%3A%20%23eee%3B%20position%3A%20fixed%3B%20bottom%3A%200%3B%20right%3A%200%3B%20height%3A%20100px%3B%20width%3A%20300px%3B%20z-index%3A%201000%3B%7D%20.urlmodal%20textarea%20%7Bwidth%3A%20100%25%3B%20height%3A%20250px%3B%7D%3C%2Fstyle%3E').appendTo('head')%3BjQuery('%3Cdiv%20class%3D%22urlmodal%22%3E%3Ch3%3ELet%5C's%20create%20a%20dataset%3C%2Fh3%3E%3Ctextarea%3EScoll%20all%20the%20way%20down%5CnClick%20%22Show%20more%20images%22%5CnScroll%20more%5CnClick%20on%20the%20images%20you%20want%20to%20remove%20from%20the%20dataset%5CnThe%20urls%20will%20appear%20in%20this%20box%20for%20you%20to%20copy.%3C%2Ftextarea%3E%3C%2Fdiv%3E').appendTo('body')%3BjQuery('%23rg').on('click'%2C%20'.rg_di'%2C%20function()%20%7BjQuery(this).toggleClass('remove')%3BupdateUrls()%3Breturn%20false%3B%7D)%3BjQuery(window).scroll(updateUrls)%3BjQuery('.urlmodal%20textarea').focus(function()%20%7BupdateUrls()%3B%20setTimeout(selectText%2C%20100)%7D).mouseup(function()%20%7Breturn%20false%3B%7D)%3Bfunction%20updateUrls()%20%7Bvar%20urls%20%3D%20Array.from(document.querySelectorAll('.rg_di%3Anot(.remove)%20.rg_meta')).map(el%3D%3EJSON.parse(el.textContent).ou)%3Bvar%20search_term%20%3D%20jQuery('.gsfi').val()%3BjQuery('.urlmodal%20textarea').val(urls.join(%22%5Cn%22))%3BjQuery('.urlmodal%20h3').html(search_term%20%2B%20%22%3A%20%22%20%2B%20urls.length)%3B%7Dfunction%20selectText()%20%7BjQuery('.urlmodal%20textarea').select()%3B%7D%7D%3Bdocument.head.appendChild(e)%3B%7D)(document.createElement('script')%2C%20'%2F%2Fcode.jquery.com%2Fjquery-latest.min.js')%7D)()
```

지금 보니 인라인이 보기 좋은거 같네요.
~백틱 세개를 1개로만 바꿔주시면 될 것 같습니당~

deep-diver committed:
Update gi2ds.md

곳곳의 북마크 -> 북마크릿으로 용어 통일
```javascript: -> `javascript 인라인 형태로 변경
북마크릿 추가 방법의 순서를 ####로 텍스트 강조함
